### PR TITLE
Fix/responsive

### PR DESCRIPTION
**Pull request by sofiatengvall**
Fixes for responsivity

## Review Checklist

- [x] All changes are described in the PR description
- [x] New feature and breaking changes have documentation
- [x] New endpoints have API tests
- [x] New frontend features have Enzyme tests
- [x] No unnecessary or debugging code
- [x] Questionable code is clearly marked with comments
- [x] Has been rebased by @tobbe-j

### CODE REVIEW

**Comment by HenrikHelenius** [comment]
Screenshot 2019-03-14 at 19 02 55

**Comment by HenrikHelenius** [comment]
overflow problem in answers

File: client/src/feedbacker-components/comment-panel/comment-panel.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@ $panel-min-width: 300px;
 
 .panel-container {
   @include display-and-animate;
-  &.hidden {
+  &.hidden, &.hide {
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment-panel/comment-panel.scss]: Why two classes? What is the difference?

File: client/src/index.js [hunk 2]
```diff
@@ -122,24 +122,34 @@ class MainView extends React.Component {
...
     }))
+
     if (this.state.surveyPanelIsHidden) {
       this.setState({ surveyButtonAnimation: true })
       setTimeout(() => {
         this.setState({ surveyButtonAnimation: false })
       }, 3000)
     }
+
+    if (window.innerWidth < 500 && !this.state.commentPanelIsHidden) {
```
Comment (HenrikHelenius) [inline client/src/index.js]: Make this a variable, so all window.innerWidth to => isMobileViewport or something, so easier to change 500 later on, replace in all places.

File: client/src/index.js [hunk 3]
```diff
@@ -122,24 +122,34 @@ class MainView extends React.Component {
...
     }))
+
     if (this.state.commentPanelIsHidden) {
       this.setState({ commentButtonAnimation: true })
       setTimeout(() => {
         this.setState({ commentButtonAnimation: false })
       }, 3000)
     }
+
+    if (window.innerWidth < 500 && !this.state.surveyPanelIsHidden) {
```
Comment (HenrikHelenius) [inline client/src/index.js]: to var

File: client/src/scss/_variables.scss [hunk 4]
```diff
@@ -27,3 +27,7 @@ $z-index-levels: (
 @function static-url($path) {
   @return url($static-url + $path);
 }
+
+// Responsivity
+
+$mobile-break-point: 500px;
```
Comment (HenrikHelenius) [inline client/src/scss/_variables.scss]: Add comment should be same as in js, specify place

**Review by HenrikHelenius — DISMISSED** [review]

File: client/src/scss/_variables.scss (hunk 4 above)
Comment (sofiatengvall) [reply client/src/scss/_variables.scss]: clarify?

File: client/src/feedbacker-components/comment/comment.js [hunk 5]
```diff
@@ -16,95 +17,130 @@ import TargetIcon from '../../assets/svg/baseline-location_searching-24px.svg'
...
+              {comment.hideName ? 'Anonymous user' : (comment.username || 'Anonymous user')}
+            </div>
+            {this.opLabel(op, comment.userId)}
+          </div>
+          <div className={css('time-target-container')}>
+            <Moment
+              className={css('timestamp')}
+              date={comment.time}
+              format="D.MM.YYYY HH.mm"
+              add={{ hours: 2 }} // REMOVE WHEN SWITCHING TO POSTGRES IN PROD
```
Comment (tobbe-j) [inline client/src/feedbacker-components/comment/comment.js]: The add(2) should be removed from here and next line

File: client/src/scss/_variables.scss [hunk 6]
```diff
@@ -23,3 +23,11 @@ $z-index-levels: (
   4: 400000,
   5: 500000,
 );
+
+@function static-url($path) {
```
Comment (tobbe-j) [inline client/src/scss/_variables.scss]: Should be removed from here, already exists in _fucntions.scss

**Review by tobbe-j — CHANGES_REQUESTED** [review]
See comments

File: client/src/scss/_variables.scss (hunk 4 above)
Comment (HenrikHelenius) [reply client/src/scss/_variables.scss]: // Breakpoint in pixels same as in js check in file: x.js

**Review by tobbe-j — APPROVED** [review]
Looks good

**Pull request merged**